packages/core/src/components/stepper/stepper.tsx

```diff
@@ -0,0 +1,113 @@
+import { Component, Host, Prop, Watch, h } from '@stencil/core'
+import DOMPurify from 'dompurify'
+
+import { isMaxTablet } from '../../utils/screens'
+
+@Component({
+  tag: 'atom-stepper',
+  styleUrl: 'stepper.scss',
+  shadow: true,
+})
+export class AtomStepper {
+  @Prop({ mutable: true }) steps:
+    | string
+    | {
+        title: string
+        completed: boolean
+      }[] = []
+  @Prop() activeStep: number
+  @Prop() stepTitle?: string
+  @Prop() disabledUnrenderedStep?: boolean = true
+
+  private parsedSteps: {
+    title: string
+    completed: boolean
+  }[] = []
+
+  @Watch('steps')
+  parseSteps(newVal) {
+    this.parsedSteps = typeof newVal === 'string' ? JSON.parse(newVal) : newVal
+  }
+
+  componentWillLoad() {
+    if (typeof this.steps === 'string') {
+      this.parsedSteps = JSON.parse(this.steps)
+    } else {
+      this.parsedSteps = this.steps
+    }
+  }
+
+  private readonly isCompleted = (completed: boolean) =>
```

Comment:
As it doesn't return a boolean, you can change to something like: addCompletedClass/shouldAddCompletedClass etc

Reply:
Nice! I chose to use `addCompletedClass`

Made: https://github.com/juntossomosmais/atomium/pull/726/commits/8e6b6de2af6f8c8dc7cedbd93d8d154026d0aa0d

Reply:
Nice! I chose to use `addCompletedClass`

https://github.com/juntossomosmais/atomium/pull/726/commits/8e6b6de2af6f8c8dc7cedbd93d8d154026d0aa0d